src/assets/scss/components/_oktaSigninWidget.scss

```diff
@@ -166,14 +179,29 @@ sign-up-modal {
     }
 
     &:last-of-type {
-
-      // We don't include password as it's the only field on it's step and has the password requirements below it.
-      input[type=text]:not(.chzn-search input),
-      .o-form-input-error,
-      .chzn-container a {
+      // Usually the last fieldset has rounded bottom borders
+      // ".chzn-container a" is for select boxes
+      input[type=text], .o-form-input-error, .chzn-container a {
```

Comment:
`input[type=text]:not(.chzn-search input)` is needed otherwise the search box inside country field has rounded borders too, which we don't want.